components/VideoBitrate.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState, useEffect } from 'react';
+import { Box, Text } from 'grommet';
+import { Participant, Stream } from '@telnyx/video';
+
+import { TelnyxRoom } from 'hooks/room';
+
+function VideoBitrate({
+  participantId,
+  streamKey,
+  getStatsForParticipantStream,
+}: {
+  participantId: Participant['id'];
+  streamKey: Stream['key'];
+  getStatsForParticipantStream: TelnyxRoom['getWebRTCStatsForStream'];
+}) {
+  const [bitrate, setBitrate] = useState(0);
+  const [currentBytesReceived, setCurrentBytesReceived] = useState(0);
+  const [previousBytesReceived, setPreviousBytesReceived] = useState(0);
+
+  useEffect(() => {
+    const interval = setInterval(async () => {
+      try {
+        const stats = await getStatsForParticipantStream(
+          participantId,
+          streamKey
+        );
+
+        let inboundRTPVideo: any;
+
+        if (stats['receivers'] && stats['receivers']['video']) {
+          inboundRTPVideo = stats['receivers']['video'];
+
+          Object.keys(inboundRTPVideo).forEach((item) => {
+            const hasVideoStream = item.match(/RTCInboundRTPVideoStream/gim);
+            if (hasVideoStream) {
+              setCurrentBytesReceived(inboundRTPVideo[item]['bytesReceived']);
+            }
+          });
+        }
+      } catch (error) {
+        setBitrate(0);
+        setCurrentBytesReceived(0);
+        setPreviousBytesReceived(0);
+        throw error;
+      }
+    }, 1000);
+
+    return () => clearInterval(interval);
+  }, [participantId, streamKey, getStatsForParticipantStream]);
+
+  useEffect(() => {
+    if (currentBytesReceived === 0) {
+      setBitrate(0);
+      return;
+    }
+
+    if (previousBytesReceived === 0) {
+      setBitrate(8 * currentBytesReceived);
+      setPreviousBytesReceived(currentBytesReceived);
+      return;
+    }
+
+    setBitrate(8 * (currentBytesReceived - previousBytesReceived));
+    setPreviousBytesReceived(currentBytesReceived);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Do we need to disable this here?

Reply:
No. I've include a condition inside the `useEffect` that returns if there's no change in bytes receives. That allows for this line to be removed and to add `setPreviousBytesReceived` as a dependency. 

I also removed the condition that would check if `currentBytesReceived` is `0`; bytes received value is incremental.

Resolved in [c0bd3f7](https://github.com/team-telnyx/telnyx-meet/pull/58/commits/c0bd3f7eef8519c32e919f37b321696a65da0018)